cmd/frontend/backend/mocks.go

```diff
@@ -3,10 +3,9 @@ package backend
 import (
 	"context"
 
-	opentracing "github.com/opentracing/opentracing-go"
-
 	"github.com/sourcegraph/sourcegraph/cmd/frontend/db"
 	"github.com/sourcegraph/sourcegraph/internal/actor"
+	"github.com/sourcegraph/sourcegraph/internal/trace/ot"
```

Comment:
I think using `ot` as the package name is too minimal:
```suggestion
	ot "github.com/sourcegraph/sourcegraph/internal/trace/opentracing"
```

Reply:
`opentracing` conflicts with `github.com/opentracing/opentracing-go`, which is still in use in some packages. `selectivetracing` seemed too long, so I settled on the short `ot`, because the package itself is a small wrapper. Can change this latter if it causes too much confusion.